config/environments/production.rb

```diff
@@ -61,4 +61,7 @@
 
   # Send deprecation notices to registered listeners
   config.active_support.deprecation = :notify
+
+  # Custom Session Store config to allow gathers.staging.ensl.org
+  config.session_store :cookie_store, key: '_ENSL_session_key', expire_after: 30.days.to_i, domain: "gathers.ensl.org"
```

Comment:
Line is too long. [118/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.